omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/PersonResource2_2Test.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs2_2;
+
+import org.junit.Before;
+import org.openmrs.Person;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+import org.openmrs.module.webservices.rest.web.v1_0.RestTestConstants2_2;
+
+public class PersonResource2_2Test extends BaseDelegatingResourceTest<PersonResource2_2, Person> {
+	
+	@Before
+	public void before() throws Exception {
+	}
+	
+	@Override
+	public Person newObject() {
+		return Context.getPersonService().getPersonByUuid(getUuidProperty());
+	}
+	
+	@Override
+	public String getDisplayProperty() {
+		 return "Mr. Horatio Test Hornblower";
+	}
+	
+	@Override
+	public String getUuidProperty() {
+		return RestTestConstants2_2.PERSON_UUID;
+	}
+	
+	@Override
+	public void validateFullRepresentation() throws Exception {
+		super.validateFullRepresentation();
+		assertPropEquals("gender", getObject().getGender());
```

Comment:
This is repetition, the call to super.validateFullRepresentation() avove has already validated all the existing fields, therefore you should only validate the new one after that.